core-api/src/main/java/com/optimizely/Optimizely.java

```diff
@@ -0,0 +1,1184 @@
+/****************************************************************************
```

Comment:
This file shouldn't exist. The Optimizely class is in the path: core-api/src/main/java/com/optimizely/ab/Optimizely.java